_posts/2021-02-19-26-rln-relay.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: post
+name:  "Privacy-preserving economic spam protection in Waku v2 with Rate Limiting Nullfiers"
+title:  "Privacy-preserving economic spam protection in Waku v2 with Rate Limiting Nullfiers"
+date:   2021-02-25 12:00:00 +0800
+author: staheri14
+published: true
+permalink: /Waku-v2-RLN-Relay
+categories: research
+summary: This post is going to give you an overview of how spam protection can be achieved in Waku Relay through rate-limiting nullifiers. We will cover a summary of spam-protection methods in centralized and p2p systems, and the solution overview and details of the economic spam-protection method. The open issues and future steps are discussed in the end.
+image: 
+discuss: 
```

Comment:
TODO update with link, I usually figure out URL and post to Discuss first with link to it

Reply:
I need to publish the rln-relay post, then open a post in https://forum.vac.dev/ with a link to the rln-relay post, and then add the link of the forum post inside the rln-relay post, right?

Reply:
You can do that, or you can compute URL from permalink and post it before it exists to discuss, then link that in post :P

Reply:
I also thought about what you said, but wasn't sure about putting a link to a non-existing post :D 
I have made the post on the https://forum.vac.dev/ but it is pending for approval